_plugins/jekyll_get.rb

```diff
@@ -0,0 +1,74 @@
+require 'json'
+require 'hash-joiner'
+require 'open-uri'
+require 'hmac_authentication'
+
+
+module Jekyll_Get
```

Comment:
Oh, this was pulled in directly?  Should we just get rid of the line in the `Gemfile` then that I noted above?